spec/game.test.js

```diff
@@ -0,0 +1,107 @@
+const { describe } = require('jest-circus');
+const Game = require('../src/game');
+
+beforeEach(() => {
+  game = new Game();
+});
+
+describe('Game', () => {
+  test('has an empty array of frames', () => {
+    expect(game.frames).toEqual([]);
+  });
+
+  test('adds a frame to the array', () => {
+    const mockFrame = { first_roll: null, second_roll: null, score: 0 };
+    game.newFrame(mockFrame);
+    expect(game.frames.length).toEqual(1);
+  });
+});
+
+describe('Strikes', () => {
+  test('strike returns true', () => {
+    const mockFrame = { first_roll: 10, second_roll: 'x', score: 10 };
+    expect(game.isStrike(mockFrame)).toEqual(true);
+  });
+
+  test('spare is not a strike', () => {
+    const mockFrame = { first_roll: 4, second_roll: 6, score: 10 };
+    expect(game.isStrike(mockFrame)).not.toEqual(true);
+  });
+});
+
+describe('Spares', () => {
+  test('spare returns true', () => {
+    const mockFrame = { first_roll: 4, second_roll: 6, score: 10 };
+    expect(game.isSpare(mockFrame)).toEqual(true);
+  });
+});
+
+describe('if first_roll exceeds 10', () => {
```

Comment:
I like that you've included tests like these in order to catch exceptions when playing in the command line.